KubeArmor/policy/policy.go

```diff
@@ -31,19 +31,46 @@ func (p *ServiceServer) ContainerPolicy(c context.Context, data *pb.Policy) (*pb
 
 		if policyEvent.Object.Metadata.Name != "" {
 
-			p.UpdateContainerPolicy(policyEvent)
-
-			res.Status = 1
-
+			present := policyCheck(policyEvent.Object.Metadata.Name)
+			updated := p.UpdateContainerPolicy(policyEvent)
+
+			if policyEvent.Type == "ADDED" {
+				if updated {
+					res.Status = 2
+				}
+				if !updated {
+					res.Status = 4
+					return res, nil
+				}
+				if updated && present {
+					res.Status = 3
+					return res, nil
+				}
```

Comment:
```suggestion
				if updated {
					res.Status = ADDED
				}
				if !updated {
					res.Status = NOCHANGE
				}
				if updated && present {
					res.Status = CONFIGURED
				}
				return res, nil
```

Use protobuf enums for defining Status

https://github.com/grpc/grpc/blob/master/doc/health-checking.md
https://protobuf.dev/programming-guides/enum/

suggested by @DelusionalOptimist